README.md

```diff
@@ -1 +1,61 @@
-# capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Designing a Responsive page Using HTML&CSS
+
+
+> Description the project.
+
+
+## Built With
+
+- html&css
+- no Frameworks
+- Flexbox
+- linter
+
```

Comment:
- [ ] Please record a video presenting the features of the project and note that the maximum length of the video is 5 minutes.
![Screenshot from 2022-12-02 21-47-08](https://user-images.githubusercontent.com/6615394/205395694-6620aa04-98a1-4d9f-9586-79409d341a4a.png)

Reply:
@codecaiine i had let a video in my readme file

Reply:
@ilyassatlassi  You are right! I found it.